cicd/crabserver_pypi/Dockerfile

```diff
@@ -25,7 +27,7 @@ WORKDIR /build
 COPY cicd/crabserver_pypi/ .
 # Getting wmcore version from wmcore_requirements.txt and install with `pip install --no-deps`
 RUN pip install -r requirements.txt
-RUN req=($(./requirementsParse.py wmcore_requirements.txt)) \
+RUN req=($(grep -v '^\s*#' < wmcore_requirements.txt)) \
```

Comment:
for dummies like me, can you add a one line comment to indicate what it does ? I am puzzled by `\s`

Reply:
Done.
(sorry, I am force-pushing with new comment lines make GH comment point to new changes instead of old one).